guacamole-docker/bin/start.sh

```diff
@@ -317,6 +317,10 @@ END
         "ldap-search-bind-password" \
         "$LDAP_SEARCH_BIND_PASSWORD"
 
+    set_optional_property       \
+        "ldap-user-search-filter" \
```

Comment:
Ever so minor nit-pick, but, in keeping with the style on the rest of the document, please line up the `\` at the end of the lines, here.

Reply:
np, aligned it first and then changed the second line indent :-(
I'm with you on clean code style.